src/openrct2-ui/interface/Window.cpp

```diff
@@ -805,13 +805,20 @@ void WindowAlignTabs(WindowBase* w, WidgetIndex start_tab_id, WidgetIndex end_ta
 
     for (i = start_tab_id; i <= end_tab_id; i++)
     {
+        auto& widget = w->widgets[i];
         if (!WidgetIsDisabled(*w, i))
         {
-            auto& widget = w->widgets[i];
             widget.left = x;
             widget.right = x + tab_width;
             x += tab_width + 1;
         }
+        else
+        {
+            // Workaround to not have two widgets share the same space otherwise WindowFindWidgetFromPoint
+            // can potentially return the disabled one which causes issues.
```

Comment:
Please add the PR ID here, so it's easier to trace back when someone bumps into this later on:

```suggestion
            // Workaround #20535: Avoid disabled wisgets from sharing the same space as active ones, otherwise
            // WindowFindWidgetFromPoint could return the disabled one, causing issues.
```

clang-format may not agree on the length.